crates/node/src/roles/worker.rs

```
@@ -279,7 +279,9 @@ where
 {
     // hack to suppress repeated logging of schema registrations
     // todo: Fix it
-    tracing::subscriber::with_default(NoSubscriber::new(), || schemas.overwrite(schema_updates))?;
+    let _ = tracing::subscriber::with_default(NoSubscriber::new(), || {
+        schemas.overwrite(schema_updates)
+    });
```

Why is it ok to swallow potential overwrite errors here?